src/mixins/actionText.js

```diff
@@ -59,6 +67,13 @@ export default {
 			 * @type {Event}
 			 */
 			this.$emit('click', event)
+
+			if (this.closeAfterClick) {
+				const parent = GetParent(this, 'Actions')
+				if (parent) {
```

Comment:
```suggestion
				if (parent && parent.closeMenu) {
```

Never too careful ;)